app/Http/Controllers/Client/UserController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers\Client;
+
+use App\Http\Controllers\Controller;
+use App\Models\Booking;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class UserController extends Controller
+{
+
+    public function index()
+    {
+        if (Auth::check()) {
+            $user = Auth::user();
+            $booking = Booking::where('user_id', '=', $user->id)
+                ->orderBy('created_at', 'DESC')
+                ->get();
+
+            return view('functions.user_detail', compact('user', 'booking'));
+        } else return view('errors.404');
+    }
+
+    public function edit($id)
+    {
+        if (Auth::check()) {
+            $user = Auth::user();
+            $booking = Booking::where('user_id', '=', $user->id)
+                ->orderBy('created_at', 'DESC')
+                ->get();
+
+            return view('functions.user_edit', compact('user', 'booking'));
+        } else return view('errors.404');
+    }
+
+    public function update(Request $request, $id)
+    {
+        $user = User::find(Auth::id());
```

Comment:
```suggestion
        $user = Auth::user();
```
em đang đi đường vòng đấy :))